django/pytest.ini

```diff
@@ -1,7 +1,7 @@
 [pytest]
 DJANGO_SETTINGS_MODULE = thunderstore.core.settings
 norecursedirs = static var htmlcov .pytest_cache .mypy_cache .vscode static_built __pycache__ node_modules
-addopts = --reuse-db --nomigrations
+addopts = --reuse-db --migrations
```

Comment:
Are we sure migrations need to be run _every_ time? As far as I know migrations only need to be run once (to enable the extension) after the test DB has been created. So with the `--reuse-db` flag, you should only run with the `--migrations` flag when doing `--create-db` as well.

Reply:
Yep, you are correct. We don't need to force it, I've fixed this in the upcoming commit